python/paddle/optimizer/lr.py

```diff
@@ -573,19 +573,16 @@ class NaturalExpDecay(LRScheduler):
             ...     scheduler = paddle.optimizer.lr.NaturalExpDecay(learning_rate=0.5, gamma=0.1, verbose=True)
             ...     sgd = paddle.optimizer.SGD(learning_rate=scheduler)
             ...     sgd.minimize(loss)
-            ...
             >>> exe = paddle.static.Executor()
             >>> exe.run(start_prog)
             >>> for epoch in range(20):
             ...     for batch_id in range(5):
             ...         out = exe.run(
             ...             main_prog,
-            ...             feed={
-            ...                 'x': np.random.randn(3, 4, 5).astype('float32'),
-            ...                 'y': np.random.randn(3, 4, 5).astype('float32')
-            ...             },
-            ...             fetch_list=loss.name)
-            ...         scheduler.step()    # If you update learning rate each step
+            ...             feed={'x': np.random.randn(3, 4, 5).astype('float32'), 'y': np.random.randn(3, 4, 5).astype('float32')},
```

Comment:
```suggestion
            ...             feed={'x': np.random.randn(3, 4, 5).astype('float32'), 'y': np.random.randn(3, 4, 5).astype('float32'),},
```

这里格式化后一行太长了明显不合适，添加 magic trailing comma 后重新格式化

Reply:
收到

Reply:
已更新，请再review一下，谢谢